### PR TITLE
[IMP] refactor intro

### PR DESCRIPTION
**Pull request by legalsylvain**
- [x] describe more precisely openupgrade and openupgradelib.
- [x] describe more precisely old and new OpenUpgrade design
- [x] specifies that the version number corresponds to the target version. (a lot of issue are related to users that try to migrate a database from 11 to 12 using openupgrade 11...
- [x] add images for single and multi steps migration
- [x] Add main reasons to realize a migration
- [x] mention alternative to Openupgrade. (Odoo SA migration vs create from scratch + imports).
- [x] mention that OpenUpgrade is recommended by odoo S.A. for migrating odoo community edition instances.

To review you can : 
- read the current intro [here](https://oca.github.io/OpenUpgrade/intro.html)
- read the proposed intro [here](https://github.com/legalsylvain/OpenUpgrade/blob/documentation-IMP-intro/docsource/intro.rst).

### CODE REVIEW

**Comment by legalsylvain** [comment]
this one is ready for review. 
CC : @bguillot, @etobella.

File: docsource/intro.rst [hunk 1]
```diff
@@ -1,28 +1,103 @@
 Introduction
 ============
 
+What is OpenUpgrade
+-------------------
+
 Odoo is an open source business application suite and development platform.
```
Comment (vdewulf) [inline docsource/intro.rst]: ==> open core instead of open source? (if we want to be honest)

File: docsource/intro.rst [hunk 2]
```diff
@@ -1,28 +1,103 @@
...
 This project, *OpenUpgrade*, aims to provide an Open Source upgrade path for
 Odoo. This is a community initiative, as the open source version of Odoo
```
Comment (vdewulf) [inline docsource/intro.rst]: This is a community initiative (under the OCA umbrella) + link to oca website?

File: docsource/intro.rst [hunk 3]
```diff
@@ -1,28 +1,103 @@
...
-does not support migrations from one major release to another. Instead,
-migrations are part of a support package sold by Odoo SA. Note that the name
-of the project refers to the old name of Odoo, *OpenERP*.
+Community Edition does not support migrations from one major release to another.
```
Comment (vdewulf) [inline docsource/intro.rst]: Odoo Community Edition (CE)
==> add the acronym because used later

File: docsource/intro.rst [hunk 4]
```diff
@@ -1,28 +1,103 @@
...
+
+    * the **framework** which is a set of a few Odoo patches to ensure that upgrading to a major version can be performed
+    * The **database analysis** which lists the differences between one version of Odoo and the next
+    * The **migration scripts** which contains database transformation scripts for each installed Odoo module.
+
+* `openupgradelib <https://github.com/OCA/openupgradelib>`_ contains a library
+  with helper functions. It can be used in the migration of any Odoo module
+
+Note that the name of the project refers to the old name of Odoo, *OpenERP*
+and that the OpenUpgrade logo includes the ant present in the *OpenERP* logo.
```
Comment (vdewulf) [inline docsource/intro.rst:28]: Where is this logo? I don't know it!

File: docsource/intro.rst [hunk 5]
```diff
@@ -1,28 +1,103 @@
...
+  in https://github.com/OCA/OpenUpgrade contain copies (or forks in Git terminology)
+  of the Odoo main project, but with extra commits that include the framework,
+  and the analysis and the migration scripts for each module.
+
+* Since Openupgrade 14.0, the branches contain
+
+    * a module ``openupgrade_framework`` which brings together all the patches of odoo
+    * a module named ``openupgrade_scripts`` which contains analysis and migration scripts.
+
+Why migrate to a more recent major version
```
Comment (vdewulf) [inline docsource/intro.rst:62]: you could add Daniel's video on this topic from OCA Days 2020: 
https://youtu.be/u7_k8lCZeY0

File: docsource/intro.rst [hunk 6]
```diff
@@ -1,28 +1,103 @@
...
+------------------------------------------
+
+* `Security holes <https://www.cvedetails.com/vulnerability-list/vendor_id-16543/Odoo.html>`_
+  are regularly found in Odoo.
+  Only the latest versions are patched, so older versions are left with security breeches that can be exploited.
```
Comment (vdewulf) [inline docsource/intro.rst:67]: the 3 latest versions are patched by Odoo S.A.

File: docsource/images/oca-modules-evolution.png
Comment (vdewulf) [inline docsource/images/oca-modules-evolution.png:1]: Add the axes X and Y and legend (blue, red...)

File: docsource/intro.rst [hunk 7]
```diff
@@ -1,28 +1,103 @@
...
 
-The branches in the first project contain the framework, as well as the
-database analysis and the migration scripts.
+To migrate from one major version to another, there are alternatives to using OpenUpgrade :
 
-The second project contains a library with helper functions. It
-can be used in the migration of any Odoo module.
+1. Odoo S.A.'s service (https://upgrade.odoo.com/).
+   Note that the migration scripts are **not Open Source** and
+   are available are only available if you subscribe to Odoo Enterprise Edition.
```
Comment (vdewulf) [inline docsource/intro.rst]: dubble "are" in the sentence

File: docsource/intro.rst [hunk 8]
```diff
@@ -1,28 +1,103 @@
...
+2. Export data (products, customers) from your old instance, install Odoo in a
+   recent version, and import data into the new version.
+   Note that in this case, **you won't recover all the data** in your new instance,
+   but only some of it.
```
Comment (vdewulf) [inline docsource/intro.rst]: but only some of it (in particular, you won't have the history and links between data, like the invoices for all customers for example).

**Review by vdewulf — CHANGES_REQUESTED** [review]

File: docsource/intro.rst (hunk 4 above)
Comment (legalsylvain) [reply docsource/intro.rst:28]: https://oca.github.io/OpenUpgrade/

File: docsource/intro.rst (hunk 1 above)
Comment (legalsylvain) [reply docsource/intro.rst]: ;-) done.

File: docsource/intro.rst (hunk 2 above)
Comment (legalsylvain) [reply docsource/intro.rst]: Done.

File: docsource/intro.rst (hunk 3 above)
Comment (legalsylvain) [reply docsource/intro.rst]: Indeed. done.

File: docsource/intro.rst (hunk 5 above)
Comment (legalsylvain) [reply docsource/intro.rst:62]: Good Idea ! done.

File: docsource/intro.rst (hunk 6 above)
Comment (legalsylvain) [reply docsource/intro.rst:67]: done, with a link to the Odoo policy.
https://www.odoo.com/documentation/master/administration/maintain/supported_versions.html#what-s-the-support-status-of-my-odoo

File: docsource/intro.rst (hunk 7 above)
Comment (legalsylvain) [reply docsource/intro.rst]: fixed.

File: docsource/intro.rst (hunk 8 above)
Comment (legalsylvain) [reply docsource/intro.rst]: added.

File: docsource/images/oca-modules-evolution.png
Comment (legalsylvain) [reply docsource/images/oca-modules-evolution.png:1]: I added a legend. I find it quite explicit : 

image

it's just to indicate a growth trend. but i can remove it, if it's not understandable, or if it doesn't add anything.

**Comment by legalsylvain** [comment]
thanks @vdewulf for your review. I applied your suggestion. let me know if there are some other points to fix.

**Review by vdewulf — APPROVED** [review]
Thanks for integrating my comments.

**Review by etobella — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**